src/styles/sidebar/components/annotation.scss

```diff
@@ -49,6 +49,15 @@
     }
   }
 
+  &__content {
+    // Prevent long strings of character text, e.g. URLs, from breaking out
+    // of container horizontally.
+    // See: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/
+    word-wrap: break-word;
```

Comment:
The `overflow-wrap: break-word` rule already exists in [markdown-view.scss](https://github.com/hypothesis/client/blob/4ed9c67cda755ad9d5f3d7beaa4a7ec9b174a45a/src/styles/sidebar/components/markdown-view.scss#L9). In testing locally, that works as expected in Safari. It doesn't work in Chrome on Firefox however with the examples on that support thread.

Adding `word-break: break-word` in `markdown-view.scss` resolves the issue. Given the subtle issues around differences in meaning between very similar-sounding properties and differences in behavior across browsers, it might be wise to extract this into a mixin where we can document that.

The other issue to resolve is whether these properties should be set on the markdown view and tag list (which currently doesn't have any wrapping rules AFAIK) or in a wrapper `<div>` as here. I personally prefer to put these rules closer to the user-defined content so they those components still work as expected if they are ever re-used in a different context. The downside is that if we add any new components that display user-defined content we'll need to remember to add wrapping rules to those.

Reply:
If adding these or a subset of these to `MarkdownView` fixes the problem, then I concur that that is where they belong—thus we can eliminate the extra wrapping element, which was a bit of a hack. I can mixin-i-fy the rules, too.